phoenix-core/src/main/java/org/apache/phoenix/exception/SQLExceptionCode.java

```diff
@@ -384,6 +367,12 @@ public SQLException newException(SQLExceptionInfo info) {
     INVALID_IMMUTABLE_STORAGE_SCHEME_AND_COLUMN_QUALIFIER_BYTES(1137, "XCL37", "If IMMUTABLE_STORAGE_SCHEME property is not set to ONE_CELL_PER_COLUMN COLUMN_ENCODED_BYTES cannot be 0"),
     INVALID_IMMUTABLE_STORAGE_SCHEME_CHANGE(1138, "XCL38", "IMMUTABLE_STORAGE_SCHEME property cannot be changed from/to ONE_CELL_PER_COLUMN "),
     CANNOT_SET_GUIDE_POST_WIDTH(1139, "XCL39", "Guide post width can only be set on base data tables"),
+
+    INVALID_DATA_TYPE_CAST(1141, "XCL41", "Invalid data type cast "),
```

Comment:
Can you change the message to "cannot modify column because of invalid data type cast"

Reply:
Ok, Thanks for your review.